package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Fri May 11 10:31:10 UTC 2018 - jlopez@suse.com
+
+- Partitioner: fix buttons to abort and to go back
+  (part of fate#318196).
```

Comment:
I don't guess so. It is incredible, ins't it?

Reply:
On the other hand, it would be nice to mention here the fix done about reprobing and proposal calculation. It's not related to the buttons of the partitioner and it's quite buried in this PR.